### DISCUSSION TITLE
Expert System

### DISCUSSION THREAD

**Discussion by masman-22** [community/community#77109] [Programming Help]
### Body

Hello everyone! i am a student who is in need of help with a project.

I have been trying to understand how to build an expert system using python and nothing seems to work out. Any help and guidelines would be appreciated please.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by rayannott** [comment]
Hi @masman-22!
I'm happy to help, but could you please clarify what an expert system is?
